projects/1.16/assets/minecraft-earth-mod/minecraft_earth_mod/lang/zh_cn.json

```
@@ -1 +1,190 @@
+  "entity.minecraft_earth_mod.sunset_cow": "日落牛",
+  "entity.minecraft_earth_mod.stormy_chicken": "暴风鸡",
+  "entity.minecraft_earth_mod.spotted_pig": "斑点猪",
+  "entity.minecraft_earth_mod.sooty_pig": "煤烟猪",
+  "entity.minecraft_earth_mod.skewbald_chicken": "白花斑鸡",
+  "entity.minecraft_earth_mod.skeleton_wolf": "骷髅狼",
+  "entity.minecraft_earth_mod.rocky_sheep": "岩石羊",
+  "entity.minecraft_earth_mod.rainbow_sheep": "彩虹羊",
+  "entity.minecraft_earth_mod.pinto_cow": "花斑牛",
+  "entity.minecraft_earth_mod.pink_moobloom": "粉红色哞花",
+  "entity.minecraft_earth_mod.pink_footed_pig": "粉蹄猪",
+  "entity.minecraft_earth_mod.pie_bald_pig": "花斑猪",
+  "entity.minecraft_earth_mod.pale_pig": "苍白猪",
+  "entity.minecraft_earth_mod.muddy_pig": "泥猪",
+  "entity.minecraft_earth_mod.mottled_pig": "杂斑猪",
+  "entity.minecraft_earth_mod.moolip": "哞金香",
+  "entity.minecraft_earth_mod.moobloom": "哞花",
+  "entity.minecraft_earth_mod.mob_of_me": "我的生物",
+  "entity.minecraft_earth_mod.midnight_chicken": "午夜鸡",
+  "entity.minecraft_earth_mod.merl": "Merl",
+  "entity.minecraft_earth_mod.melon_golem": "西瓜傀儡",
+  "entity.minecraft_earth_mod.lobber_zombie": "投手僵尸",
+  "entity.minecraft_earth_mod.jolly_llama": "快乐羊驼",
+  "entity.minecraft_earth_mod.horned_sheep": "角羊",
+  "entity.minecraft_earth_mod.gold_crested_chicken": "金冠鸡",
+  "entity.minecraft_earth_mod.glowing_squid": "发光鱿鱼（测试）",
+  "entity.minecraft_earth_mod.furnace_golem": "熔炉傀儡",
+  "entity.minecraft_earth_mod.fancy_chicken": "华丽鸡",
+  "entity.minecraft_earth_mod.dyed_cat": "染色猫",
+  "entity.minecraft_earth_mod.dried_muddy_pig": "干燥的泥猪",
+  "entity.minecraft_earth_mod.cookie_cow": "曲奇牛",
+  "entity.minecraft_earth_mod.cluckshroom": "咯菇",
+  "entity.minecraft_earth_mod.brule_cow": "奶油牛",
+  "entity.minecraft_earth_mod.bronzed_chicken": "青铜鸡",
+  "entity.minecraft_earth_mod.boulder_zombie": "攀岩僵尸",
+  "entity.minecraft_earth_mod.bone_spider": "骨蜘蛛",
+  "entity.minecraft_earth_mod.ashen_cow": "灰白牛",
+  "entity.minecraft_earth_mod.amber_chicken": "琥珀鸡",
+  "entity.minecraft_earth_mod.albino_cow": "白化牛",
+  "entity.minecraft_earth_mobs.muddy_pig": "泥猪",
+  "block.minecraft_earth_mod.stone_tappable": "石头可点击物",
+  "block.minecraft_earth_mod.spruce_tappable": "云杉木可点击物",
+  "block.minecraft_earth_mod.ruby_shop_block": "红宝石交易方块",
+  "block.minecraft_earth_mod.ruby_ore": "红宝石矿石",
+  "block.minecraft_earth_mod.ruby_block": "红宝石块",
+  "block.minecraft_earth_mod.rainbow_wool": "彩虹羊毛",
+  "block.minecraft_earth_mod.rainbow_carpet": "彩虹地毯",
+  "block.minecraft_earth_mod.pink_daisy": "粉红色雏菊",
+  "block.minecraft_earth_mod.oak_tappable": "橡木可点击物",
+  "block.minecraft_earth_mod.mud_converter": "泥浆转化器",
+  "block.minecraft_earth_mod.mud": "泥浆",
+  "block.minecraft_earth_mod.grass_tappable": "草可点击物",
+  "block.minecraft_earth_mod.furnace_golem_light": "熔炉傀儡的光源",
+  "block.minecraft_earth_mod.chest_tappable": "箱子可点击物",
+  "block.minecraft_earth_mod.carved_melon": "雕刻过的西瓜",
+  "block.minecraft_earth_mod.buttercup": "金凤花",
```

```suggestion
  "block.minecraft_earth_mod.buttercup": "毛茛",
```
https://minecraft.fandom.com/zh/wiki/Minecraft_Earth:%E6%AF%9B%E8%8C%9B